language/changes/7-packages.md

```diff
@@ -0,0 +1,321 @@
+# Packages
+
+* Status: Implemented in Move 1.5
+
+## Introduction
+
+Packages are a new source feature in Move version 1.5 to allow Move programmers
+to more easily re-use code and share it across projects. Previously in order to
+build large sets of Move modules that spanned multiple projects a custom build
+script needed to be hand rolled. Moreover, with a large Move project there are
+additional generated artifacts that need to be taken into account, e.g., source
+maps, generated documentation, and ABIs. The Move Package System (MPS) allows
```

Comment:
I went with it because it sounded good as I was writing, but now that you mention it, I think it's probably best to remove for now and have a larger team discussion on names.